data_steward/deid/aou.py

```diff
@@ -277,16 +276,12 @@ def initialize(self, **args):
         Press.initialize(self, **args)
         LOGGER.info(f"BEGINNING de-identification on table:\t{self.tablename}")
 
-        age_limit = args['age_limit']
+        age_limit = args.get('age_limit', 89)
```

Comment:
I think it would be good to define a constant `MAX_AGE=89` and reuse it so we won't have to change it in multiple places if we need to.

Reply:
It's a good idea.  I wasn't sure if it was scope creep.  I will make this change.